packages/frontend/src/components/wallet-migration/metrics.js

```diff
@@ -88,3 +91,23 @@ export const resetUserState = () => {
     }
     return rudderanalytics.reset();
 };
+
+export function accountIdToHash(accountId) {
+    const hash = sha256.create();
+    hash.update(accountId);
+    return hash.hex();
+};
```

Comment:
Per our discussion: unless there's a good reason to persist each hash, I think you should drop the functions that get/set localstorage and only use this method to compute the hash each time.